packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-run-sample/main/nodejs/testapp.md

```diff
@@ -0,0 +1,32 @@
+## Steps to test the sample app
+
+1. If not already done, set up your Okta org by completing these steps: [Set up your Okta org (for password factor only use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-password-factor-only-use-cases).
+1. If not already done,
+   [download and set up the sample app](/docs/guides/oie-embedded-common-download-setup-app/nodejs/main/).
+1. Locate the sample apps solution file in the following path:
+`...\okta-auth-js\samples\generated\express-embedded-auth-with-sdk`
+1. Open `express-embedded-auth-with-sdk` using Visual Studio or the editor of your choice.
+1. Add a `testenv` configuration file. See [Option 1: Configuration file](/docs/guides/oie-embedded-common-download-setup-app/nodejs/main/#option-1-configuration-file) for further information.
+1. From the terminal, navigate to the `express-embedded-auth-with-sdk` sample directory and run the application: `npm run start`. This process sets the environment variables and starts the web server. Check the console to see that the environment variables are set.
+1. Add an `okta.yaml` configuration file. See [Option 1: YAML configuration file](/docs/guides/oie-embedded-common-download-setup-app/aspnet/main/#option-1-configuration-file) for more information on how to configure and where to place the configuration file.
```

Comment:
is the yaml file from the .net sample? it's not needed in nodejs sample

Reply:
Yes, sorry, used the .net content as a base. Thanks, will clean up.